### PR TITLE
simplify code

### PR DESCRIPTION
**Pull request by retokromer**
should fix https://github.com/amiaopensource/vrecord/issues/231
(merge after https://github.com/amiaopensource/vrecord/pull/232)

### CODE REVIEW

File: vrecord [hunk 1]
```diff
@@ -462,23 +462,9 @@ lookup_standard(){
...
-    esac
-}
-
-lookup_qctoolsxml(){
-    case "${1}" in
-        "Yes")
-            ;;
-        "No")
-            ;;
+        "Yes")|"No") ;;
```
Comment (aeschweik) [inline vrecord]: The `"Yes")|"No")` syntax doesn't seem to work (I get the response: ``line 467: syntax error near unexpected token `|'`` from Terminal), but if I change it to `"Yes"|"No")` it stops complaining.

Comment (retokromer) [reply vrecord]: Yes, of course, my typo… Thanks a lot!

**Comment by retokromer** [comment]
Typo fixed. It was correct on https://github.com/amiaopensource/vrecord/issues/231#issuecomment-359259372

**Comment by retokromer** [comment]
Thank you, @aeschweik, for reviewing.

**Pull request merged**